docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc

```diff
@@ -43,14 +43,16 @@ Hide on login page::
 If this identity provider should be hidden in login pages when the user is authenticating in the scope of the organization.
 
 Redirect when email domain matches::
-If members should be automatically redirected to the identity provider when their email domain matches the domain set to the identity provider.
+If members should be automatically redirected to the identity provider when their email domain matches the domain set to the identity provider. If the domain is set to `Any`, members whose e-mail matches *any* of the organization domains will be redirected to the identity provider.
+
+If the org is linked with multiple identity providers, the organization authenticator prioritizes the provider that matches the e-mail of the user for automatic redirection. If none is found, it tries to locate one whose domain is set to `Any`.
```

Comment:
nit:
Maybe rewrite as follows?

```
The organization authenticator chooses the identity provider for automatic redirection in the following order:
* by email domain match
* If no identity provider is found, an identity provider whose domain is set to `Any`.
```

Or if you want to not change it too much (not a native english speaker myself - sorry):

```
If the organization is linked with multiple identity providers, the organization authenticator prioritizes the provider that matches the email domain of the user for automatic redirection. If none is found, the organization authenticator tries to locate an identity provider whose domain is set to `Any`.
```

Reply:
Thanks, will adjust for better clarity.